scapy/layers/dhcp.py

```diff
@@ -210,11 +211,13 @@ def getfield(self, pkt, s):
     118: IPField("subnet-selection", "0.0.0.0"),
     124: "vendor_class",
     125: "vendor_specific_information",
+    128: IPField("tftp-server-ip-address", "0.0.0.0"),
```

Comment:
Could you use `_` instead of `-` to be consistent with your modifications? Thanks. Otherwise, this looks good to be merged.